ttnn/cpp/ttnn/operations/matmul/device/matmul_op.hpp

```diff
@@ -12,6 +12,8 @@
 #include "ttnn/tensor/tensor_utils.hpp"
 #include "ttnn/types.hpp"
 
+#include "ttnn/operations/experimental/ccl/ccl_op_fusion.hpp"
```

Comment:
It'd be good to not use experimental in non-experimental ops.

In other files you use `
#include "ttnn/cpp/ttnn/operations/ccl/kernel_common/worker_sync_utils.hpp"`

Would it be possible to move what's necessary to something that's not experimental?

Reply:
The files that are used by other non-experimental files, have now been moved to a non-experimental folder.